Meadowlark-js/backends/meadowlark-mongodb-backend/test/middleware/SecurityMiddleware.test.ts

```diff
@@ -0,0 +1,313 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { FrontendRequest, FrontendResponse, newFrontendRequest, newFrontendResponse } from '@edfi/meadowlark-core';
+import { newFrontendRequestMiddleware } from '@edfi/meadowlark-core/src/handler/FrontendRequest';
+import { newPathComponents } from '@edfi/meadowlark-core/src/model/PathComponents';
+import { securityMiddleware } from '../../src/security/SecurityMiddleware';
+import * as OwnershipSecurity from '../../src/repository/OwnershipSecurity';
+
+jest.setTimeout(40000);
+
+describe('given the upsert where response already posted', () => {
+  let result;
+  const mongoClienttMock = {};
+
+  const frontendRequest: FrontendRequest = {
+    ...newFrontendRequest(),
+    action: 'upsert',
+    middleware: {
+      ...newFrontendRequestMiddleware(),
+      pathComponents: { ...newPathComponents() },
+    },
+  };
+
+  const frontendResponse: FrontendResponse = {
+    ...newFrontendResponse(),
+    statusCode: 200,
+  };
+
+  beforeAll(async () => {
+    result = await securityMiddleware({ frontendRequest, frontendResponse }, mongoClienttMock as any);
+  });
+
+  it('should not do anything', async () => {
```

Comment:
This phrase "should not do anything" was probably copied from somewhere else, and it kind of makes sense. But I have to really think about it. I think most of these `it` statements need to have clearer text.

```suggestion
  it('should return the original arguments without modification', async () => {
```